WordPress/Classes/ViewRelated/Settings/SupportViewController.m

```diff
@@ -250,7 +251,7 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
     }
 
     if (section == SettingsSectionActivityLog) {
-        return 5;
+        return 6;
```

Comment:
These have been already there, so it might not fit into this PR, but changing this ^ and other magical numbers to constants could increase readability especially for reviews.